test-data/unit/check-isinstance.test

```diff
@@ -1443,6 +1443,28 @@ else:
 [builtins fixtures/isinstancelist.pyi]
 
 
+[case testIssubclasDestructuringUnions]
+from typing import Union, List, Tuple, Dict, Type
+def f(x: Union[Type[int], Type[str], Type[List]]) -> None:
```

Comment:
@ilevkivskyi what do you see happening? everything seems fine in my forked repo, when I merged this branch into master.